src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -683,17 +683,31 @@ namespace static_llm {
 StatefulLLMPipeline::StatefulLLMPipeline(
     const std::filesystem::path& models_path,
     const ov::genai::Tokenizer& tokenizer,
-    const std::string&,
+    const std::string& device,
     const ov::AnyMap& config
 ) : LLMPipelineImplBase(tokenizer,
                         utils::from_config_json_if_exists(models_path)) {
-
-    auto model = genai::utils::singleton_core().read_model(models_path / "openvino_model.xml", {}, config);
-    ModelConfigDesc model_desc = get_modeldesc_from_json(models_path / "config.json");
     ov::AnyMap properties = config;
+    const auto use_blobs = pop_or_default(properties, "USE_BLOBS", false);
+    if (use_blobs) {
+        auto blob_path = pop_or_default(properties, "BLOB_PATH", std::string{});
```

Comment:
```suggestion
        auto blob_path = pop_or_default(properties, "BLOB_PATH", std::filesystem::path{});
```
and drop `.string()` below

Reply:
Below in the log we still need to convert it to string, so I would leave it like that. Plus it's mostly the same logic as we already do in `setupAndImportModels()`